### PR TITLE
Hide equivalent values for RSK network

### LINKED ISSUE #2091: Wrong rate for RSK
### Description of the Issue

MyCrypto API is taking SBTC as "SuperBitcoin" (or another coin) instead of "SmartBitcoin" from RSK.
SBTC is pegged 1:1 to BTC, so that must be the rate.
Users may think that they are paying far less than what they are actually paying (in USD).

This is used in `common/api/rates.ts`.

### Steps to Reproduce

a) Select RSK network and check any fee conversion to USD, or
b) Check `https://proxy.mycryptoapi.com/cc/price?fsym=ETH&tsyms=SBTC` conversion or `https://proxy.mycryptoapi.com/cc/price?fsym=BTC&tsyms=SBTC`

### PR DESCRIPTION
**Pull request by alepc253**
Closes #2091

### Description

RSK currency symbol is SBTC "Smart Bitcoin" that crashes to SBTC "Super Bitcoin". This generates mistakes in rate conversions because MyCrypto API returns "Super Bitcoin" rate for SBTC.

### Changes

* Addition of new configuration parameter `hideEquivalentValues` (optional boolean) that avoids showing rate conversions and, consequently, misunderstandings operating with RSK network.

### Steps to Test

1. Select RSK network
2. Go to "View&Send" / "Contracts" tab and prepare a transaction.
3. Check that no USD values are shown.
4. Check that "Equivalent Values" box shows a message about not showing values for this unit.

### CODE REVIEW

**Comment by Azarielle** [comment]
[Added to Asana](https://app.asana.com/0/763966770926912/779983315837065)

**Comment by alepc253** [comment]
Hi guys, any news about this? it's really important to us to have this addressed because of the cost misinformation

File: common/components/BalanceSidebar/EquivalentValues.tsx [hunk 1]
```diff
@@ -191,6 +191,10 @@ class EquivalentValues extends React.Component<Props, State> {
           <div className="text-center">
             <h5 style={{ color: 'red' }}>{translate('EQUIV_VALS_TESTNET')}</h5>
           </div>
+        ) : network.hideEquivalentValues ? (
+          <div className="text-center">
+            <h5 style={{ color: 'red' }}>{translate('EQUIV_VALS_UNSUPPORTED_UNIT')}</h5>
```
Comment (ConnorBryan) [inline common/components/BalanceSidebar/EquivalentValues.tsx]: @alepc253 Move this to the colocated .scss file.

**Comment by ConnorBryan** [comment]
@alepc253 
1. Move out inline styling from EquivalentValues.
2. Fix merge conflict in EquivalentValues.tsx by using the new `feature` format (configSelectors and configMetaSelectors)
3. Same thing in FeeSummary
4. Re-add AQUA to the networks list, only use `import * as types` for consistency, and get rid of the changes to the other networks.

That should be it.

File: common/components/BalanceSidebar/EquivalentValues.tsx (hunk 1 above)
Comment (alepc253) [reply common/components/BalanceSidebar/EquivalentValues.tsx]: I'm not sure about this... I added this code replicating how it is solved for `network.isTestnet`, does it make sense to be changed?

**Pull request closed without merging**